linter/linter.go

```diff
@@ -168,6 +173,8 @@ func Run() bool {
 		}
 	}
 
+	// 5. No errors occurred, and we've printed all warnings. Decide whether
+	// we need to print anything else, then exit.
 	if linter.anyWarning && !linter.Warnings {
```

Comment:
Minor thing, but this will say warnings were suppressed even if there were only warnings for sequences not specified in `--sequences`.

Reply:
Oh, uh, also, `linter.anyWarning` doesn't seem to take into account `seqResults.AnyWarning`?